tt_metal/jit_build/data_format.cpp

```diff
@@ -245,15 +186,10 @@ std::vector<DataFormat> get_unpack_dst_formats(
         TT_FATAL(unpack_to_dest_mode.size() == NUM_CIRCULAR_BUFFERS, "unpack_to_dest_mode vector must have 32 elements");
     }
 
-    DataFormat pack_format = get_pack_data_format(output_formats, intermed_formats);
-    ExpPrecision input_precision = get_data_exp_precision(input_formats);
-
     std::vector<DataFormat> unpack_dst_format;
 
-    const bool en_unpack_tf32 = fp32_dest_acc_en && (tt::is_all_fp32_formats(input_formats) || (input_precision == ExpPrecision::B));
-    DataFormat unpack_cond_dst_format = en_unpack_tf32 ? DataFormat::Tf32 : unpack_conditional_dst_format;
```

Comment:
This line (and the one above it) may or may not be necessary. The unpack_conditional_dst_format is also set to Tf32 in `genfiles.cpp` so it may be redundant, but this one also adds the OR condition for the precision to be B format.